### PR TITLE
OC-1231 Complete

### PR DESCRIPTION
**Pull request by iwoodferren**

### CODE REVIEW

File: web/src/main/java/org/akaza/openclinica/control/managestudy/ResolveDiscrepancyServlet.java [hunk 1]
```diff
@@ -93,11 +93,21 @@ public Page getPageForForwarding(DiscrepancyNoteBean note, boolean isCompleted)
             }
             // UpdateStudyEvent?event_id=12&ss_id=12
         } else if ("itemdata".equalsIgnoreCase(entityType) || "eventcrf".equalsIgnoreCase(entityType)) {
-            if (currentRole.getRole().equals(Role.MONITOR) || !isCompleted) {
+            /*if (currentRole.getRole().equals(Role.MONITOR) || !isCompleted) {
                  return Page.VIEW_SECTION_DATA_ENTRY_SERVLET;
                 // ViewSectionDataEntry?eventDefinitionCRFId=&ecId=1&tabId=1&studySubjectId=1
             } else {
                 return Page.ADMIN_EDIT_SERVLET;
+            }*/
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/control/managestudy/ResolveDiscrepancyServlet.java:101]: if this code is no longer needed, remove it completely. You don't need to comment it out

File: web/src/main/webapp/WEB-INF/jsp/submit/initialDataEntryNw.jsp [hunk 2]
```diff
@@ -387,7 +387,8 @@ function initmb(){
 
 }
 
-window.onload = initmb;
+window.onload = initmb;<c:if test='${popUpURL != ""}'>
```
Comment (svadla-oc) [inline web/src/main/webapp/WEB-INF/jsp/submit/initialDataEntryNw.jsp:390]: move the if statement to next line

File: web/src/main/java/org/akaza/openclinica/control/managestudy/ResolveDiscrepancyServlet.java [hunk 3]
```diff
@@ -93,11 +93,21 @@ public Page getPageForForwarding(DiscrepancyNoteBean note, boolean isCompleted)
...
+
+            //POSSIBLE SOLUTION TO OC-1231
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/control/managestudy/ResolveDiscrepancyServlet.java:103]: is it a possible solution or final solution? :)

Comment (iwoodferren) [reply web/src/main/java/org/akaza/openclinica/control/managestudy/ResolveDiscrepancyServlet.java]: hahaha final.

**Comment by svadla-oc** [comment]
👍

**Pull request merged**